src/benchmark_runner.cc

```diff
@@ -232,7 +232,7 @@ class BenchmarkRunner {
     // Base decisions off of real time if requested by this benchmark.
     i.seconds = i.results.cpu_time_used;
     if (b.use_manual_time) {
-      i.seconds = i.results.manual_time_used;
+      i.seconds = b.time->Cost(i.results.manual_time_used);
```

Comment:
this line makes me think that `b.time` should instead be `b.manual_time` to make it clear it's set from 'manual time'.

Reply:
`b.time` is used with regular time unit measurements too, as per @LebedevRI's advice: it is used for all time measurements, so maybe I should specify `ManualCost` to be more explicit just for this one member (as that is what it is used for).